dev/com.ibm.ws.jdbc/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -768,6 +768,8 @@ portNumber=Port number
 portNumber.desc=Port on which to obtain database connections.
 prefetch=Prefetch
 prefetch.desc=JDBC driver property: prefetch.
+preparedStatementCacheQueries=Prepared Statement Cache Queries
```

Comment:
Change 
"Prepared Statement Cache Queries" 
to sentence case
"Prepared **s**tatement **c**ache **q**ueries"